src/client/kinto_http.rs

```diff
@@ -41,15 +55,14 @@ pub async fn get_records(
     let resp = reqwest::get(&url).await?;
     let timestamp = resp
         .headers()
-        .get("etag")
-        .unwrap()
-        .to_str()
-        .unwrap()
-        .to_string();
-    let size = resp.headers().get("content-length").unwrap();
+        .get("etag").ok_or_else(|| KintoError::Error {name: "no etag error".to_owned()});
+
+    println!("Timestamp : {:?}", timestamp);
+
+    let size = resp.headers().get("content-length").ok_or_else(|| KintoError::Error {name: "no content error".to_owned()});
```

Comment:
We don't really use the `size` apart from a print. Defaulting to `-1` should be enough IMO